### PR TITLE
[AND-2905] Fix start mute for banner/MREC ads.

### PR DESCRIPTION
**Pull request by HuFangshuai**

### CODE REVIEW

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java [hunk 1]
```diff
@@ -77,13 +77,17 @@ public Bundle build() {
 
   public static AdConfig adConfigWithNetworkExtras(Bundle networkExtras) {
     AdConfig adConfig = new AdConfig();
-    adConfig.setMuted(true); // start muted by default.
```
Comment (palrahul) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java:80]: @HuFangshuai wanted to confirm that if no network extras set to request, by not explicitly setting any muted flag are we leaving it to the creative setting ?
It seems to be different than when network extras is set but EXTRA_START_MUTED flag is not added, in which case. we are invoking setMute(false)

Comment (HuFangshuai) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java:80]: fixed, declare unmute settings for ads.

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java [hunk 2]
```diff
@@ -77,13 +77,18 @@ public Bundle build() {
...
+    adConfig.setMuted(false); // Unmute fullscreen ads by default
```
Comment (alexander-gubov) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java:80]: actually, it's false by default and will be combined with creative settings

Comment (HuFangshuai) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java:80]: yes, here can make ads mute logic run to the same result on adapter side.

**Pull request merged**